### PR TITLE
Vector Integer Arithmetic

### PR DESCRIPTION
**Pull request by yulong-plct**
delete some case about vmul[h/hu/hsu] and fix scan-assembler-times error.

### CODE REVIEW

File: rvv-intrinsic-generator/rvv_intrinsic_gen/inst.py [hunk 1]
```diff
@@ -53,20 +53,20 @@ def gen(g):
   else:
     decorators = IntrinsicDecorators(g.has_tail_policy)
 
-  g.start_group("Configuration-Setting and Utility Functions")
+  """g.start_group("Configuration-Setting and Utility Functions")
```
[the remark this answers is not in the document]
Comment (yulong-plct) [reply rvv-intrinsic-generator/rvv_intrinsic_gen/inst.py:56]: I noticed that. It's my fault. I will finish the rest and submit it together today. @Incarnation-p-lee

File: intrinsic-test.sh [hunk 2]
```diff
@@ -28,6 +28,7 @@ make clean
 ./configure --prefix="$PWD/opt-rvv64gc" --with-arch=rv64gcv --with-abi=lp64d --with-isa-spec=20191213 --with-sim=qemu
 make -j $(nproc) build-sim SIM=qemu
 
+rm -rf gcc/gcc/testsuite/gcc.target/riscv/rvv/gnu-auto-generated/
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:31]: suggest to file another patch for unrelated changes here, otherwise LGTM.

Comment (yulong-plct) [reply intrinsic-test.sh]: I deleted

**Review by Incarnation-p-lee — APPROVED** [review]
